rpa-home/modules/ROOT/pages/automation-userrolesandpermissions-privilegesbymodule.adoc

```diff
@@ -0,0 +1,358 @@
+= List of Privileges by Module
+
+RPA Manager has the following privileges, which can be assigned to individual users.
+
+To open any module, you always require the privilege for which the name ends in *Open*.
+
+Privileges that end with *Administration* authorize a user to create, change, and delete all types of objects in the corresponding module.
+If the authorization only applies to a single type of object in a module, the corresponding privileges end with  *Create*, *Edit*, and *Delete*.
+
+Privileges named *Global <...> View* authorize a user to see all the processes in the relevant view. A user who does not have this privilege can only see processes for which they belong to the project team.
+
+If you assign a role to a user, the user receives all the privileges contained in the role. These privileges are not displayed in the *Privileges* field in the user's profile.
+
+
+
+
+== Privileges of category *System*
+
+[cols="1,2"]
+|===
+|*Privilege* |*Permitted Action*
+
+|*Logfile Download*
```

Comment:
Probably should mention that this will be removed in the future and has no real use right now

Reply:
Removed from documentation